pages/submissions/[id]/index.tsx

```diff
@@ -83,6 +84,12 @@ const TaskListPage = ({
         </div>
         <div className="govuk-grid-column-one-third">
           <div className={s.sticky}>
+            <Link href={`/submissions/${params.id}/printable`}>
```

Comment:
We also want the Link on finished submissions right?

pages/people/{personId}/submissions/{submissionId}

Reply:
maybe - i wanna see if we can get by without for now. it seems like they just want the form's questions, not the submission info per se.

Reply:
so, once a form has been submitted, they'd no longer need to print it off

Reply:
Right, it seems fine for preliminary purpose of allowing people to print off the form.

From what I understand part of the purpose is the printing is to print out a completed form that can then be sent to someone.

The goal is that a form can be completed on the system as normal and if for some reason some other organisation needs a paper copy the printed form with resident information can all be printed off. No filling out paperwork by hand!

As a start I'm ok with this though as it isn't clear how much functionality is required.